### PR TITLE
fix: Tidy up fetch with init examples on Request and Fetch pages

### PR DESCRIPTION
**Pull request by bsmth**
### Description

I'm reflecting some changes that landed in the DOM examples repo already.

### Motivation

Get rid of some redundant info in the request, add in some practical options for cache control preferences, linking to Cache-Control header.

### Related issues and pull requests

- [x] Related changes: https://github.com/mdn/dom-examples/pull/270
- [x] Original issue: https://github.com/mdn/dom-examples/issues/151

### CODE REVIEW

File: files/en-us/web/api/fetch/index.md [hunk 1]
```diff
@@ -295,50 +295,44 @@ fetch(myRequest)
...
+In our [Fetch Request with init example](https://github.com/mdn/dom-examples/tree/main/fetch/fetch-request-with-init) (see [Fetch Request init live](https://mdn.github.io/dom-examples/fetch/fetch-request-with-init)) we do the same thing except that we pass in an _options_ object when we invoke `fetch()`.
+In this case, we can set a {{httpheader("Cache-Control")}} value to indicate what kind of cached responses we're okay with:
 
 ```js
 const myImage = document.querySelector("img");
+const reqHeaders = new Headers();
 
-const myHeaders = new Headers();
-myHeaders.append("Accept", "image/jpeg");
+// a cached response is okay unless it's more than a week old
```
Comment (teoli2003) [inline files/en-us/web/api/fetch/index.md:305]: NIT:
```suggestion
// A cached response is okay unless it's more than a week old
```

File: files/en-us/web/api/fetch/index.md [hunk 2]
```diff
@@ -295,50 +295,44 @@ fetch(myRequest)
...
-  method: "GET",
-  headers: myHeaders,
-  mode: "cors",
-  cache: "default",
+const options = {
+  headers: reqHeaders,
 };
 
-const myRequest = new Request("flowers.jpg");
+// pass init as an "options" object with our headers
```
Comment (teoli2003) [inline files/en-us/web/api/fetch/index.md:312]:
```suggestion
// Pass init as an "options" object with our headers.
```

File: files/en-us/web/api/request/request/index.md [hunk 3]
```diff
@@ -142,59 +141,54 @@ fetch(myRequest)
...
+In our [Fetch Request with init example](https://github.com/mdn/dom-examples/tree/main/fetch/fetch-request-with-init) (see [Fetch Request init live](https://mdn.github.io/dom-examples/fetch/fetch-request-with-init)) we do the same thing except that we pass in an _options_ object when we invoke `fetch()`.
+In this case, we can set a {{httpheader("Cache-Control")}} value to indicate what kind of cached responses we're okay with:
 
 ```js
 const myImage = document.querySelector("img");
+const reqHeaders = new Headers();
 
-const myHeaders = new Headers();
-myHeaders.append("Content-Type", "image/jpeg");
+// a cached response is okay unless it's more than a week old
```
Comment (teoli2003) [inline files/en-us/web/api/request/request/index.md:151]:
```suggestion
// A cached response is okay unless it's more than a week old.
```

**Review by teoli2003 — APPROVED** [review]
A few NITs, so pre-approving.

**Comment by bsmth** [comment]
Thank you!

**Pull request merged**